### PR TITLE
[DX-875] Dashboard 5.2.4 change log

### PR DESCRIPTION
**Pull request by andyo-tyk**
Draft change log for 5.2.4 (Dashboard)

https://tyktech.atlassian.net/browse/DX-875

https://deploy-preview-3695--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-dashboard/release-notes/version-5.2/

Note that this is a draft: link to Docker image to be added.

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md [hunk 1]
```diff
@@ -14,6 +14,82 @@ Minor releases are supported until our next minor comes out. There is no 5.3 sch
...
+ <li>
+ <details>
+ <summary>Poor experience when using the Open Policy Agent (OPA) editor</summary>
+
+ Fixed two UI issues with the [OPA editor]({{< ref "tyk-dashboard/open-policy-agent#using-the-open-policy-agent-in-the-dashboard" >}}) in the Tyk Dashboard to improve experience when using this feature. Scrolling beyond the end of the OPA window does not now start to scroll the API Designer window, and minimising then re-expanding the OPA editor no longer limits the text to one line.
+ </details>
+ </li>
+ <li>
+ <details>
+ <summary>Annoying bugs when setting Dashboard user access controls</summary>
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md:51]: What was the issue? Did it relate to the user being unable to configure user access or limited permissions?

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Thanks Andy, PR LGTM!!

When release date and docker image available update details. Let me know when TBC change log item added and I'll review. Added a question about the user access control bug summary

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md (hunk 1 above)
Comment (andyo-tyk) [reply tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md:51]: Settings changed unexpectedly. I don't think we need to go into the details.

**Review by dcs3spp — APPROVED** [review]

**Pull request merged**